test/sharness/t0139-swarm-rcmgr.sh

```diff
@@ -9,57 +9,73 @@ test_init_ipfs
 # swarm limit|stats should fail in offline mode
 
 test_expect_success 'disconnected: swarm limit requires running daemon' '
-    test_expect_code 1 ipfs swarm limit system 2> actual &&
-    test_should_contain "missing ResourceMgr" actual
+  test_expect_code 1 ipfs swarm limit system 2> actual &&
+  test_should_contain "missing ResourceMgr" actual
 '
 test_expect_success 'disconnected: swarm stats requires running daemon' '
-    test_expect_code 1 ipfs swarm stats all 2> actual &&
-    test_should_contain "missing ResourceMgr" actual
+  test_expect_code 1 ipfs swarm stats all 2> actual &&
+  test_should_contain "missing ResourceMgr" actual
 '
 
 # swarm limit|stats should fail in online mode by default
 # because Resource Manager is opt-in for now
 test_launch_ipfs_daemon
 
 test_expect_success 'ResourceMgr disabled by default: swarm limit requires Swarm.ResourceMgr.Enabled' '
-    test_expect_code 1 ipfs swarm limit system 2> actual &&
-    test_should_contain "missing ResourceMgr" actual
+  test_expect_code 1 ipfs swarm limit system 2> actual &&
+  test_should_contain "missing ResourceMgr" actual
 '
 test_expect_success 'ResourceMgr disabled by default: swarm stats requires Swarm.ResourceMgr.Enabled' '
-    test_expect_code 1 ipfs swarm stats all 2> actual &&
-    test_should_contain "missing ResourceMgr" actual
+  test_expect_code 1 ipfs swarm stats all 2> actual &&
+  test_should_contain "missing ResourceMgr" actual
 '
 
 # swarm limit|stat should work when Swarm.ResourceMgr.Enabled
 test_kill_ipfs_daemon
+
 test_expect_success "test_config_set succeeds" "
-  ipfs config --json Swarm.ResourceMgr.Enabled true
+  ipfs config --json Swarm.ResourceMgr.Enabled true &&
+  ipfs config --json Swarm.ResourceMgr.Limits.System.Conns 99999
```

Comment:
We should have statup test without setting `Swarm.ResourceMgr.Limits` first,  making sure `ipfs config --json Swarm.ResourceMgr.Enabled true` is enough to enable manager with the implicit defaults.
 (see previous comment in `core/node/libp2p/rcmgr.go` about panic)